desktop-exporter/app/routes/main-view.tsx

```diff
@@ -29,57 +36,64 @@ function Row({ index, style, data }: RowProps) {
 }
 
 type SidebarProps = {
-  isClosed: boolean;
+  isFullWidth: boolean;
   toggle: () => void;
 };
 
 function Sidebar(props: SidebarProps) {
-  if (props.isClosed) {
-    return (
-      <div className="sidebar closed">
-        <button
-          className="menuBtn"
-          onClick={props.toggle}
-        >
-          Expand
-        </button>
-      </div>
+  const sidebarColour = useColorModeValue("pink.100", "pink.900");
+  const { traceSummaries } = useLoaderData() as TraceSummaries;
+
+  let sidebarWidth = "80px";
+  let buttonIcon = <ArrowRightIcon />;
+  let traceList = <></>;
+
+  if (props.isFullWidth) {
+    sidebarWidth = "250px";
+    buttonIcon = <ArrowLeftIcon />;
+    traceList = (
+      <FixedSizeList
+        className="list"
+        height={500}
+        itemData={traceSummaries}
+        itemCount={traceSummaries.length}
+        itemSize={30}
+        width="100%"
```

Comment:
I believe the list may be being squeezed as it collapses. We might want to put the list into a fixed width box so the container shrinking doesn't affect its layout

Reply:
I will fix this thing in the next PR, when I style the list itself.